cmd/influx/main.go

```diff
@@ -245,6 +245,10 @@ func (c *CommandLine) executeQuery(query string) {
 		fmt.Printf("ERR: %s\n", err)
 		return
 	}
+	if results.Error() != nil {
```

Comment:
If you look at line 255 you are already checking this -- can you update the code so you are not checking the error status in two places?